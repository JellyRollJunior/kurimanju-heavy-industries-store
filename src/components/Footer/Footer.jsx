import styles from './Footer.module.css';

const Footer = () => {
  return (
    <footer className={styles.footer}>
      Created by <a className={styles.link} href="https://github.com/JellyRollJunior">JellyRollJunior</a>
      <br />
      Assets from <a className={styles.link} href="https://www.kogado.com/">Power Dolls Kogado Studios</a> and <a className={styles.link} href="https://www.theleagueofmoveabletype.com/orbitron?style=light">Matt Mcinerney</a>
    </footer>
  );
};

export { Footer };
