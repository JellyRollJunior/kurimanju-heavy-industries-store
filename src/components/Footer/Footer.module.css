.footer {
  composes: quaternaryColor from '../../styles/colors.module.css';
  grid-column: 2 / -1;
  text-align: center;

  border: 2px solid;
  border-radius: 4px;
  margin: 0 12px 12px 12px;
  padding: 18px 12px
}
