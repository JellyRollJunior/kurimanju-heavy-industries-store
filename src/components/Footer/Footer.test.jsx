import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import { Footer } from "./Footer";

describe('Footer component', () => {
    it('Renders site acknowledgements', () => {
        const { container } = render(<Footer />);

        expect(container).toMatchSnapshot();
    })
})