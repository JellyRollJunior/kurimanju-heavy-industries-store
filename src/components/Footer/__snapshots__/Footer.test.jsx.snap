// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer component > Renders site acknowledgements 1`] = `
<div>
  <footer
    class="_footer_eb428f"
  >
    Created by 
    <a
      class="_link_eb428f"
      href="https://github.com/JellyRollJunior"
    >
      JellyRollJunior
    </a>
    <br />
    Assets from 
    <a
      class="_link_eb428f"
      href="https://www.kogado.com/"
    >
      Power Dolls Kogado Studios
    </a>
     and 
    <a
      class="_link_eb428f"
      href="https://www.theleagueofmoveabletype.com/orbitron?style=light"
    >
      Matt Mcinerney
    </a>
  </footer>
</div>
`;
